server/admin_management/api/app/routers/router_guards.py

```diff
@@ -124,103 +140,66 @@ async def get_target_user_from_id(
-            raaid
+        LOGGER.debug(
+            "Dependency 'get_target_user_from_id': 'application_admin_id' "
+            + f"path param found: {raaid}."
         )
+        application_admin = application_admin_service.get_application_admin_by_id(raaid)
         if application_admin is not None:
             return TargetUser.model_validate(application_admin.user)
         else:
-            error_msg = "Parameter 'application_admin_id' {raaid} is missing or invalid."
+            error_msg = (
+                "Parameter 'application_admin_id' {raaid} is missing or invalid."
+            )
             utils.raise_http_exception(
-                error_code=ERROR_CODE_INVALID_REQUEST_PARAMETER,
-                error_msg=error_msg
+                error_code=ERROR_CODE_INVALID_REQUEST_PARAMETER, error_msg=error_msg
             )
 
     elif "access_control_privilege_id" in request.path_params:
         acpid = request.path_params["access_control_privilege_id"]
-        LOGGER.debug("Dependency 'get_target_user_from_id': 'access_control_privilege_id' " +
-                     f"path param found: {acpid}.")
-        access_control_privilege = access_control_privilege_service.get_acp_by_id(
-            acpid
+        LOGGER.debug(
+            "Dependency 'get_target_user_from_id': 'access_control_privilege_id' "
+            + f"path param found: {acpid}."
         )
+        access_control_privilege = access_control_privilege_service.get_acp_by_id(acpid)
         if access_control_privilege is not None:
             return TargetUser.model_validate(access_control_privilege.user)
         else:
-            error_msg = "Parameter 'access_control_privilege_id' {acpid} is missing or invalid."
+            error_msg = (
+                "Parameter 'access_control_privilege_id' {acpid} is missing or invalid."
+            )
             utils.raise_http_exception(
-                error_code=ERROR_CODE_INVALID_REQUEST_PARAMETER,
-                error_msg=error_msg
+                error_code=ERROR_CODE_INVALID_REQUEST_PARAMETER, error_msg=error_msg
             )
     else:
-        # from body - {user_name/user_type_code}, when grant admin access
+        # from body - {user_name/user_type_code/user_guid}, when grant admin access
         rbody = await request.json()
         LOGGER.debug(f"Dependency 'get_target_user_from_id' request body {rbody}.")
-        rb_user_name = rbody["user_name"]
-        rb_user_type_code = rbody["user_type_code"]
-        # user_guid is searched from IDIM search and passed from frontend.
-        rb_user_guid = rbody["user_guid"]
-
-        user = user_service.get_user_by_domain_and_name(
-            rb_user_type_code,
-            rb_user_name,
+
+        target_new_user = TargetUser.model_validate(
+            {
+                "user_name": rbody["user_name"],
+                "user_type_code": rbody["user_type_code"],
+                "user_guid": rbody["user_guid"],
+            }
```

Comment:
Most of the changes in this file are formatting. So here since we'll validate the user later, I don't feel we need to try to find the user from database anymore

Reply:
seems like we still need that to await request.json()